test/functional/interface_bitcoin_cli.py

```diff
@@ -72,8 +73,48 @@ def run_test(self):
             assert_equal(cli_get_info['paytxfee'], wallet_info['paytxfee'])
             assert_equal(cli_get_info['relayfee'], network_info['relayfee'])
             assert_equal(self.nodes[0].cli.getwalletinfo(), wallet_info)
+
+            # Setup to test -getinfo and -rpcwallet= with multiple wallets.
+            wallets = ['', 'Encrypted', 'secret']
+            amounts = [Decimal('59.999928'), Decimal(9), Decimal(31)]
+            self.nodes[0].createwallet(wallet_name=wallets[1])
+            self.nodes[0].createwallet(wallet_name=wallets[2])
+            w1 = self.nodes[0].get_wallet_rpc(wallets[0])
+            w2 = self.nodes[0].get_wallet_rpc(wallets[1])
+            w3 = self.nodes[0].get_wallet_rpc(wallets[2])
+            w1.walletpassphrase(password, self.rpc_timeout)
+            w1.sendtoaddress(w2.getnewaddress(), amounts[1])
+            w1.sendtoaddress(w3.getnewaddress(), amounts[2])
+
+            # Mine a block to confirm; adds a block reward (50 BTC) to the default wallet.
+            self.nodes[0].generate(1)
+
+            self.log.info("Test -getinfo with multiple wallets loaded returns no balance")
+            assert_equal(set(self.nodes[0].listwallets()), set(wallets))
+            assert 'balance' not in self.nodes[0].cli('-getinfo').send_cli().keys()
+
+            self.log.info("Test -getinfo with multiple wallets and -rpcwallet returns specified wallet balance")
+            for i in range(len(wallets)):
+                cli_get_info = self.nodes[0].cli('-getinfo').send_cli('-rpcwallet={}'.format(wallets[i]))
```

Comment:
nit: rcpwallet is an option, not a command

Reply:
Thanks, will update in #18594 or next opportunity.

Reply:
> rpcwallet is an option, not a command

Tucked the cli option vs command test fixups into the last commit of #18594 adding tests for that PR.